### ISSUE TITLE
No support for non-parallel builds (/maxcpucount=1 or "BuildInParallel")

### ISSUE THREAD

**Issue by IQpierce** [JetBrains/resharper-unity#105]
I'm working on a project with a complex set of project dependencies and build rules.

Compilation always occurred correctly in Visual Studio, but Rider (and Xamarin, FWIW) seem to interpret the rules differently. The result is that my project builds can sometimes occur in a sequence that causes the build to fail.

In Xamarin Studio I'm able to fix this problem by unchecking a checkbox which says "Allow parallel builds." I assume that it uses either the "/maxcpucount=1" msbuild parameter, or the "BuildInParallel" task parameter:
https://msdn.microsoft.com/en-us/library/bb651793.aspx

However, Rider doesn't expose a way to disable the parallel-build behavior. It would be nice if it did so.

(I tried several ways to control this behavior myself in a roundabout way, but eventually gave up... let me know if there's a clean way that I could, for instance, add the maxcpucount parameter directly to the command-line parameters of the build processes that Rider launches. But either way it would be nice to have a user-friendly way to toggle this behavior.)

**Comment by van800** [comment]
Hi! Is this issue still valid for you?
There are 2 options on Windows you may try "Use up to N processes in parallel" and "ReSharper Build".
On MAC and Linux "Use up to N processes in parallel" doesn't work and is not present in UI.
![image](https://user-images.githubusercontent.com/1482681/28014608-8ae6dae8-656d-11e7-9cc6-bd9c4ac6b879.png)
